File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/AbyssalIngot.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.Item;

/* JADX INFO: loaded from: classes3.dex */
public class AbyssalIngot extends Item {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.item_abyssal_ingot_name;
        this.idDescription = R.string.item_abyssal_ingot_description;
        this.idImage = R.drawable.abyssal_ingot;
        this.price = 1811L;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/AbyssalIngot.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.Item

class AbyssalIngot : Item() {
    override fun configureProperties() {
        idName = R.string.item_abyssal_ingot_name
        idDescription = R.string.item_abyssal_ingot_description
        idImage = R.drawable.abyssal_ingot
        price = 1811L
    }
}
